apps/edit/app.lisp

```diff
@@ -140,26 +145,27 @@
 	(vdu-load vdu buffer ox oy cx cy)
 	(view-dirty slider))
 
-(defq current_text (open-buffer msg_path))
-
 ;open the window
 (gui-add (apply view-change (cat (list window 48 16)
 	(view-pref-size (window-set-title
 		(component-connect (window-connect-close (window-connect-min
 			(window-connect-max window event_win_max) event_win_min) event_win_close) event_win_layout) 
-				(elem text_title current_text))))))
+				"")))))
 
 ;open the current buffer and set the scroll bar and textfield.
+
+(each (lambda (_) (open-buffer _)) *env_edit_auto*)
+(if (= (length text_store) 0) (open-buffer ""))
```

Comment:
(each (lambda (_) (open-buffer _)) *env_edit_auto*)

If only a single param to open-buffer, or whatever function, you can just say.

(each open-buffer *env_edit_auto*)

Reply:
Also if you has a 2 param function etc say called "takes-two"

(each takes-two seq1 seq2)

Each sequential element from each sequence is grouped and sent to a call to the function.... same with some, any, every etc. All predicate and container enum functions work this way.

Reply:
(each (lambda (_) (open-buffer _)) *env_edit_auto*)
(if (= (length text_store) 0) (open-buffer ""))

I suspect you could combine these:

(each open-buffer (if (= (length *env_edit_auto*) 0) '("") *env_edit_auto*))

?